src/core/Elsa.Abstractions/Services/Workflows/IWorkflowRegistry.cs

```diff
@@ -36,5 +36,10 @@ public interface IWorkflowRegistry
         /// Returns a single workflow blueprint that fits the specified predicate. 
         /// </summary>
         Task<IWorkflowBlueprint?> FindAsync(Func<IWorkflowBlueprint, bool> predicate, CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Returns a single workflow blueprint that fits the specified predicate. 
+        /// </summary>
+        Task<IWorkflowBlueprint?> FindActiveAsync(Func<IWorkflowBlueprint, bool> predicate, CancellationToken cancellationToken = default);
```

Comment:
I'm not sure it makes much sense for a method that finds active workflows to still require a predicate, because when you want to find "active" workflows, that implies a predicate already: the user wants to find active workflows.

I suggest removing the predicate argument. If the user needs a more specific subset, they should use the other overload that accepts a predicate, giving the user full control over the query.